### ISSUE TITLE
kernelsize default equal to 2 in UpsamplingDeconvBlock?

### ISSUE THREAD

**Issue by Yuxiang1990** [JunMa11/SegWithDistMap#1]
for example, in vnet_multi_head.py line 100, the default parameter named kernel_size in Conv3d equal to 2? It makes me confused.

**Comment by JunMa11** [comment]
Yes, the same as [downsampling](https://github.com/JunMa11/SegWithDistMap/blob/eb3d8ac09b34ad2e7f2f0e9904a74932f13feccd/code/networks/vnet_multi_head.py#L68).

**Comment by Yuxiang1990** [comment]
Hi, thanks for your share firstly,
Without tanh operation to constraint the variant out_dis in train_LA_AAAISDF.py?

**Comment by JunMa11** [comment]
It is in vnet_sdf.py `self.tanh = nn.Tanh()`

**Comment by Yuxiang1990** [comment]
May I have your suggestion about loss function named hd_loss in train_LA_HD.py line 106? what reference when you coding, thanks for your kindly help.

**Comment by JunMa11** [comment]
The TMI paper "Reducing the Hausdorff Distance in Medical Image Segmentation with Convolutional Neural Networks"
